lib/msf/core/db_manager/loot.rb

```diff
@@ -87,4 +90,39 @@ def report_loot(opts)
     ret[:loot] = loot
   }
   end
+
+  def update_loot(opts)
+    wspace = opts.delete(:workspace) || opts.delete(:wspace) || workspace
```

Comment:
Do we need to perform all of this work since `wspace` is unused, or could we simply delete the `workspace` key from `opts`?